Makefile

```diff
@@ -0,0 +1,17 @@
+IMAGE ?= w0rp/ale
+DOCKER = docker run -a stderr --rm -v $(PWD):/testplugin -v $(PWD)/test:/home "$(IMAGE)"
+
+test-setup:
+	docker images -q $(IMAGE) || docker pull $(IMAGE)
+
+test: test-setup
+	vims=$$(docker run --rm $(IMAGE) ls /vim-build/bin | grep -E '^n?vim'); \
+	if [ -z "$$vims" ]; then echo "No Vims found!"; exit 1; fi; \
+	EXIT=0; \
+	for vim in $$vims; do \
+	  $(DOCKER) $$vim '+Vader! test/*' || EXIT=$$?; \
+	done; \
+	$(DOCKER) vint -s /testplugin || EXIT=$$?; \
```

Comment:
It checks the entire repo, because of this line
```
DOCKER = docker run -a stderr --rm -v $(PWD):/testplugin -v $(PWD)/test:/home "$(IMAGE)"
```
This part in particular mounts pwd, which is the repo, into the /testplugin directory of the container. 
```
-v $(PWD):/testplugin
```

I did this because that's how the vim-testbed scripts seemed to work.